Configuration/StandardSequences/python/Reconstruction_Data_cff.py

```diff
@@ -16,25 +16,6 @@
         qTest.cleaningThresholds.append(30.)
         qTest.flags.append('HFDigi')
              
-particleFlowRecHitHF.producers[0].qualityTests.append(
```

Comment:
Hi Slavat,

I agree with you. It is better NOT to use HFDigiTime and HF timing for hit cleaning.

Shuichi.

> On Oct 1, 2015, at 5:35 AM, Halil Saka <halil.saka@cern.ch> wrote:
> 
> Hi Salavat;
> 
> Shuichi (also cc'ed) can also comment on this but I think his obsevations were in line with your comment.
> 
> Thanks
> Halil
> 
> 
>> On 01 Oct 2015, at 11:29, Salavat Abdoulline <Salavat.Abdoulline@cern.ch> wrote:
>> 
>> 
>> Michalis,
>> 
>> (adding Halil to Cc) There is quite some difference between the two "selections" below...
>> 
>> (1)
>> HFDigiTime analyses the TS pattern and sets the flag if the bulk of the signal is out of Trig.TS. This means it doesn't work at 25ns, but should be operational at 50ns.
>> 
>> (2)
>> HF timing was and is unreliable, +5/-5ns cannot work properly (loosing legitimate signal) even for high-enough signal (not only for 30 GeV, which kind of average HF RecHit energy nowadays).
>> 
>> Run 1 :
>> https://cms-cpt-software.web.cern.ch/cms-cpt-software/General/Validation/SVSuite/HCAL/750_75X_dataRun1_HLT_v1_vs_750_pre6-75X_dataRun1_v2_RelVal/Jet/RecHits/HF_Timing_vs_Energy.gif
>> 
>> Run 2:
>> http://hep.baylor.edu/cmshcal/Validation/753patch1_75X_dataRun2_HLT_withOfflineCustomisation_v0_newCond_vs_753_75X_dataRun2_HLT_frozen_v2_RelVal/Jet/RecHits/HF_Timing_vs_Energy.gif
>> 
>> 
>> Salavat
>> 
>>> On Thu, 1 Oct 2015, Michalis Bachtis wrote:
>>> 
>>> In Configuration/StandardSequences/python/Reconstruction_Data_cff.py:
>>>> @@ -16,25 +16,6 @@
>>>>        qTest.cleaningThresholds.append(30.)
>>>>        qTest.flags.append('HFDigi')
>>>>> -particleFlowRecHitHF.producers[0].qualityTests.append(
>>> Hello, So the +-5ns is historical since Run I . The question to HCAL is how do they apply the HFDigiTime criterion.
>>> If they apply a timing window this should be equivalent to the +-5 ns window that was implemented in PF for hits
>>> with E>30 GeV. I would expect that a +-5ns window could have small effect in 50ns as well. The physics question to
>>> the DPG is if they consider safe a +-5 ns window for E>30 GeV in HF hits. If they want it they can keep it in Run I,
>>> if they already do it via the severity cuts they can remove it. As I discussed several times in PPD meeting , the
>>> selection of hits in PF is something that should be decided by the DPG [as we do in ECAL] so I fully trust them to
>>> decide what they want to do
>>> �
>>> Reply to this email directly or view it on GitHub.[AEx02jPpo_TEke8z9Zd9nuTYWlTtXUZKks5o3PkwgaJpZM4GGgk3.gif]